apertium_apy/apy.py

```diff
@@ -178,11 +183,13 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    langNames = 'langNames.db'
```

Comment:
I would call this `langNamesDefault` and put it at the top of the file.

Reply:
Okay that can be done.

Reply:
Fixed it